Sources/SWBApplePlatform/AppIntentsMetadataTaskProducer.swift

```diff
@@ -58,8 +55,12 @@ final class AppIntentsMetadataTaskProducer: PhasedTaskProducer, TaskProducer {
         context.addDeferredProducer {
             let scope = self.context.settings.globalScope
             var deferredTasks: [any PlannedTask] = []
-            let buildFilesProcessingContext = BuildFilesProcessingContext(scope)
-            let swiftSources: [FileToBuild] = self.filterBuildFiles(self.sourcesBuildPhase?.buildFiles, identifiers: ["sourcecode.swift"], buildFilesProcessingContext: buildFilesProcessingContext)
+            let buildFilesProcessingContext = BuildFilesProcessingContext(self.context.settings.globalScope)
+
+            guard let configuredTarget = self.targetContext.configuredTarget, let buildPhaseTarget = configuredTarget.target as? BuildPhaseTarget else {
```

Comment:
nit: suggest you could just move this above the addDeferredProducer block and return an empty array instead of fatal-erroring